app/helpers/mascot_helper.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module MascotHelper
+  def mascot_url
+    instance_presenter.mascot&.file&.url || helpers.asset_pack_path('media/images/elephant_ui_plane.svg')
```

Comment:
I think it's missing a `full_asset_url` call.

Also I think it should just be `asset_pack_path` and not `helpers.asset_pack_paths`, and the module should `extend ActiveSupport::Concern` and `include Webpacker::Helper`

Reply:
I'm sorry I don't know ruby very well. Can you explain more?

Reply:
Actually this is fine as you changed it.